### PR TITLE
Emulator

### PR DESCRIPTION
**Pull request by todesj**

### CODE REVIEW

**Review by BraydenKO — APPROVED** [review]
Lgtm

**Review by Levi-Lesches — APPROVED** [review]
LGTM

File: lib/src/services/firebase_core.dart [hunk 1]
```diff
@@ -7,13 +8,17 @@ import "package:firebase_core/firebase_core.dart";
 /// the process, we register Firebase as a separate service that can keep track
 /// of whether it has been initialized. 
 class FirebaseCore {
+	static bool shouldUseEmulator = false;
```
Comment (Levi-Lesches) [inline lib/src/services/firebase_core.dart:11]:
```suggestion
	/// Whether the Firebase Local Emulator Suite should be used. 
	///
	/// This setting should be enabled during development ONLY. We can also use 
	/// `kDebugMode` from `flutter/foundation`, but it's also valid to test UI 
	/// or logic operations with the production database.
	static bool shouldUseEmulator = false;
	
```

**Review by Levi-Lesches — CHANGES_REQUESTED** [review]
Sorry, missed this one change. The analyzer complained about it too

**Review by Levi-Lesches — APPROVED** [review]

**Pull request merged**